### PR TITLE
module.xml properties updates to support versioning changes: "version…

### PR DESCRIPTION
**Pull request by labkey-adam**
…" -> "schemaVersion", "labkeyVersion" -> "releaseVersion", remove "consolidateScripts"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ but also make certain assumptions that you may not want to impose on your module
 [LabKey documentation](https://www.labkey.org/Documentation/wiki-page.view?name=gradleModules) for more information.
 
 ## Release Notes
+### version x.y.z
+*Released*: xxx
+(Earliest compatible LabKey version: 20.3)
+
+* Rename "version" property to "schemaVersion" and support null value
+* Rename "labkeyVersion" property to "releaseVersion"
+* Remove "consolidateScripts" property
```
Comment (labkey-susanh) [inline README.md]: Can you qualify these comments with something like "Change module properties/module.xml writer"?

File: src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy [hunk 2]
```diff
@@ -126,21 +120,17 @@ class ModuleExtension
...
-        if (modProperties.getProperty("ManageVersion") == null)
+        if (modProperties.getProperty("SchemaVersion") == null)
         {
-            if (isExternalModule)
-                modProperties.setProperty("ManageVersion", "false")
+            if (modProperties.getProperty("Version") == null)
+                modProperties.setProperty("SchemaVersion", "")  // Spring binds this as setSchemaVersion(null), which is what we want
             else
-                modProperties.setProperty("ManageVersion", "true")
+                modProperties.setProperty("SchemaVersion", modProperties.getProperty("Version"))  // For backward compatibility with old modules TODO: Remove
```
Comment (labkey-susanh) [inline src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy:133]: Do we never use want to use the SchemaVersion value? (I assume that's better handled in the code itself, but a comment to that effect would be handy here.)

**Review by labkey-susanh — CHANGES_REQUESTED** [review]

File: src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy (hunk 2 above)
Comment (labkey-adam) [reply src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy:133]: We do use it... if non-null, it gets copied into the template. This is the SchemaVersion == null (not specified in module.properties) case. Maybe I'm not understanding what you're asking for...

File: README.md (hunk 1 above)
Comment (labkey-adam) [reply README.md]: Updated

File: src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy (hunk 2 above)
Comment (labkey-susanh) [reply src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy:133]: Ah, right.  Wasn't seeing the else part.

**Review by labkey-susanh — APPROVED** [review]

**Pull request merged**